### PR TITLE
Update customer-key-manage.md

### PR DESCRIPTION
**Pull request by brdesai**
Changes to reflect disabling of new-moverequest cmdlet for local mailbox moves. This was based on the instructions from Exchange MRS team.

### CODE REVIEW

**Comment by mypil** [comment]
@JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks.

File: microsoft-365/compliance/customer-key-manage.md [hunk 1]
```diff
@@ -137,13 +137,7 @@ Get-MailboxStatistics -Identity <GeneralMailboxOrMailUserIdParameter> | fl IsEnc
...
 The IsEncrypted property returns a value of **true** if the mailbox is encrypted and a value of **false** if the mailbox is not encrypted.
 
-The time to complete mailbox moves depends on the size of the mailbox. If Customer Key hasn't completely encrypted the mailbox after 72 hours from the time you assign a new DEP, initiate a mailbox move. To do this, use the New-MoveRequest cmdlet and provide the alias of the mailbox. For example:
-  
-```powershell
-New-MoveRequest <alias>
-```
-
-For more information about this cmdlet, see [Get-MailboxStatistics](https://docs.microsoft.com/powershell/module/exchange/move-and-migration/new-moverequest?view=exchange-ps).
+The time to complete mailbox moves depends on the size of the mailbox. If Customer Key hasn't completely encrypted the mailbox after 72 hours from the time you assign a new DEP, contact Microsoft support for help. The New-MoveRequest cmdlet is no longer available for local mailbox moves. Refer to [the announcement](https://techcommunity.microsoft.com/t5/exchange-team-blog/disabling-new-moverequest-for-local-mailbox-moves/bc-p/1332141) for additional information.
```
Comment (JohanFreelancer9) [inline microsoft-365/compliance/customer-key-manage.md]:
```suggestion
The time to complete mailbox moves depends on the size of the mailbox. If Customer Key hasn't completely encrypted the mailbox after 72 hours from the time you assign a new DEP, contact Microsoft support for help. The New-MoveRequest cmdlet is no longer available for local mailbox moves. Refer to [this announcement](https://techcommunity.microsoft.com/t5/exchange-team-blog/disabling-new-moverequest-for-local-mailbox-moves/bc-p/1332141) for additional information.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - minor edit.

**Comment by mypil** [comment]
@brdesai - please merge the commit on the copy/edit review done by JohanFreelancer9 or make the necessary changes on the PR following the suggestions made. Thank you.

**Comment by brdesai** [comment]
@mypil The suggested change has been committed.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by UserAdminDEUS** [comment]
Y'all better fix my IEM on my phone before I unsecured all of y'all

**Comment by UserAdminDEUS** [comment]
Just because I show you one way of me getting in does it mean I can still
do it fix my phone

**Comment by mypil** [comment]
@KCCross - PR has been copy edited and is ready for final review. 

Thank you.

cc: @kenwith

**Pull request merged**